scripts/ts-api-extractor/src/adapters/out/ts-morph/parsers/file-parser.ts

```
@@ -0,0 +1,150 @@
+/**
+ * Props extraction parser module
+ *
+ * Parser layer extracts ParsedComponent[] from a source file.
+ * Legacy extractProps() remains for backward compatibility.
+ */
+import type { ModuleDeclaration, Node, SourceFile, Symbol } from 'ts-morph';
```

Biome (2.4.7)

[error] Do not shadow the global "Symbol" property.

(lint/suspicious/noShadowRestrictedNames)